list/source/GridListImageItem.js

```diff
@@ -146,12 +154,26 @@
 				this.selectedChanged();
 				this.imageSizingChanged();
 				this.centeredChanged();
+				this.placeholderChanged();
 			};
 		}),
 
 		/**
 		* @private
 		*/
+		placeholderChanged: function(){
```

Comment:
I don't think it's necessary to define this here. Any logic related to the internals of `enyo.Image` should be encapsulated within that kind, and I think there is a way to specify a `placeholder` while maintaining the use of `src` vs. background-image.